test/Garnet.test/RespConfigTests.cs

```diff
@@ -0,0 +1,241 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Linq;
+using Garnet.common;
+using Garnet.server;
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using StackExchange.Redis;
+
+namespace Garnet.test
+{
+    [TestFixture]
+    public class RespConfigTests
+    {
+        GarnetServer server;
+        private string memorySize = "17g";
+        private string indexSize = "64m";
+        private string objectStoreLogMemorySize = "17m";
+        private string objectStoreHeapMemorySize = "32m";
+        private string objectStoreIndexSize = "8m";
+
+        [SetUp]
+        public void Setup()
```

Comment:
Verify that this works:
* Start server with log size X, insert keys
* Take checkpoint and kill (similar to `SeSaveRecoverTest`)
* Recover server with smaller & larger size Y
* Verify keys